image/checkpoint/build-image.sh

```diff
@@ -14,7 +14,7 @@ function image::build() {
     cp ${BOOTKUBE_ROOT}/image/checkpoint/Dockerfile ${TEMP_DIR}
     cp ${BOOTKUBE_ROOT}/image/checkpoint/checkpoint-install.sh ${TEMP_DIR}
     cp ${BOOTKUBE_ROOT}/image/checkpoint/checkpoint-pod.yaml ${TEMP_DIR}
-    sed -i "s#{{ REPO }}:{{ TAG }}#${IMAGE_REPO}:${VERSION}#" ${TEMP_DIR}/checkpoint-pod.yaml
+    sed -i.bk "s#{{ REPO }}:{{ TAG }}#${IMAGE_REPO}:${VERSION}#" ${TEMP_DIR}/checkpoint-pod.yaml
```

Comment:
This is literally so it can [work](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/sed.1.html) on Darwin. I can omit that change and keep it to myself.